src/main/resources/org/jenkinsci/plugins/workflow/flow/GlobalDefaultFlowDurabilityLevel/global.jelly

```diff
@@ -13,5 +13,5 @@
             </select>
         </f:entry>
     </f:section>
-    <f:section title=""></f:section>  <!-- Visual boundary -->
+    <br/><br/>
```

Comment:
Be careful—you might be inside a `<tr>` here, in which case `<br/>` is illegal content.